### PR TITLE
OHM-928: getting started section

### PR DESCRIPTION
**Pull request by BMartinos**
This PR includes a new documentation section for getting started with the OpenHIM

By following the described steps, the user should be able to get the OpenHIM up and running quite quickly

### CODE REVIEW

File: docs/tutorial/mediator.md [hunk 1]
```diff
@@ -2,5 +2,379 @@
...
+
+---
+
+### Step 4 - Adding Mediator Heartbeat
+
+To add a mediator heartbeat, import the `activateHeartbeat` method form `openhim-mediator-utils`. This function takes in the `openhimConfig` option set to instantiate as well as the mediator's `urn` within that object. Therefore, to keep the file neat let's import the `urn` from `mediatorConfig` and add this variable to the openhimConfig object. Instantiate the `activateHeartbeat` method within the `app.listen`. The `index.js` file should now resemble this:
```
Comment (bradsawadye) [inline docs/tutorial/mediator.md]: import the `activateHeartbeat` method from

File: docs/getting-started/configuration.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+id: configuration
+title: Configuration
+sidebar_label: Configuration
+---
+
+Once you have successfully installed the OpenHIM core and console and you are able to successfully access the console at [http://localhost:9000](http://localhost:9000) we can proceed to create a basic channel and client so that we can send a test request through the OpenHIM core successfully
+
+A **Client** is usually some system that you want to able to send request to the OpenHIM. Setting up a **client** allows the OpenHIM to authenticate requests. A **Channel** defines a path that a request will take through the OpenHIM. It describes one more **routes** for the request to be forwarded to, which **clients** are allowed to use this **channel**, which requests are to be directed to this **channel** and many more options that allow you to controls what happens for a particular request.
```
[the remark this answers is not in the document]
Comment (BMartinos) [reply docs/getting-started/configuration.md]: done in 34c6633

File: docs/getting-started/configuration.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- Click on the blue "**+ Channel**" button to open a popup modal box where you will supply the **Channel** details **OR** click on one of the existing **Channels** to open up the popup modal with the **Channels'** saved details.
+- Supply all the required fields and click the blue "**Save changes**" button when completed.
+
+The two _most_ important fields to supply are the **URL Pattern** field and the **Allowed roles and clients** field. The **URL Pattern** field describes which incoming requests should be send down this **channel**. It does this by looking at the URL of the incoming request and testing if it matches the RegEx that you supply in this field. Note, only the first matched **channel** that is found receives the request for processing. The **Allowed roles and clients** field identifies which **clients** are allowed to send requests to this **channel**. If a request matches a **channel** but the **client** system is not specified in this field or a **role** containing that the **client** belongs to is not specified in this field then the request will be denied access to the **channel**.
```
[the remark this answers is not in the document]
Comment (BMartinos) [reply docs/getting-started/configuration.md]: done in 34c6633

File: docs/getting-started/install.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+## Install Core and Console
+
+Once we have all our prerequisites completed we are now ready to spin up our OpenHIM instance using Docker. Open up a new terminal window and ensure that you are within the directory where the `docker-compose.yml` file is located and execute the below command:
+
+```bash
+docker-compose up -d
+```
+
+**NB!** the user you are executing the above command with needs to be an administrator for the docker-compose script to run successfully
```
[the remark this answers is not in the document]
Comment (BMartinos) [reply docs/getting-started/install.md]: done in 3d1eecf

File: docs/tutorial/mediator.md (hunk 1 above)
Comment (bradsawadye) [reply docs/tutorial/mediator.md]: fixed in commit to follow

File: docs/tutorial/mediator.md [hunk 5]
```diff
@@ -2,5 +2,379 @@
...
+
+COPY . /app
+
+RUN npm install
+
+CMD npm start
+EXPOSE 3000
+```
+
+In your IDE move back to the `index.js` file. Here enter teh following script to setup a basic express server listening on port 3000.
```
[the remark this answers is not in the document]
Comment (bradsawadye) [reply docs/tutorial/mediator.md]: fixed in commit to follow

File: docs/getting-started/prerequisites.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+To easily get started with implementing the OpenHIM Core and Console and to start playing with the various features we can follow the below steps to get a local OpenHIM environment setup.
+
+The below method of spinning up the OpenHIM makes use of [Docker containers](https://www.docker.com/resources/what-container) that allows for easier deployments of the OpenHIM 
+
+We will need to install some software that will allow us to spin up the OpenHIM with ease. We will using [Docker](https://www.docker.com/) which is a container management service to create our OpenHIM instance.
+
+## Install Docker
+
+Docker is the engine that will be running our entire local OpenHIM instance by using the publicly available OpenHIM core and console images. To install Docker please follow the relevant steps based on your operating system you are using as described in the below link
```
[the remark this answers is not in the document]
Comment (BMartinos) [reply docs/getting-started/prerequisites.md]: done in 764888e

File: docs/getting-started/prerequisites.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+id: prerequisites
+title: Prerequisites
+sidebar_label: Prerequisites
+---
+
+To easily get started with implementing the OpenHIM Core and Console and to start playing with the various features we can follow the below steps to get a local OpenHIM environment setup.
+
+The below method of spinning up the OpenHIM makes use of [Docker containers](https://www.docker.com/resources/what-container) that allows for easier deployments of the OpenHIM 
+
+We will need to install some software that will allow us to spin up the OpenHIM with ease. We will using [Docker](https://www.docker.com/) which is a container management service to create our OpenHIM instance.
```
[the remark this answers is not in the document]
Comment (BMartinos) [reply docs/getting-started/prerequisites.md]: done in 764888e

**Pull request merged**